packages/core/src/products/details/redux/reducer.js

```diff
@@ -544,6 +548,43 @@ const merchantsLocations = (
   }
 };
 
+const outfits = (
+  state = INITIAL_STATE.outfits,
+  /* istanbul ignore next */ action = {},
+) => {
+  switch (action.type) {
+    case actionTypes.GET_PRODUCT_OUTFITS_REQUEST:
+      return {
+        isLoading: {
+          ...state.isLoading,
+          [action.payload.productId]: true,
+        },
+        error: INITIAL_STATE.outfits.error,
```

Comment:
Shouldn't this reset only the error for the requested productId?

Reply:
We have followed the same logic of existing similar reducers, for consistency sake.

Reply:
Can you show another reducer that has the same logic? That is because it is a bug the way it is right now. It will clear all errors for all products instead of the product that was requested. If there are other reducers that do the same thing, they need to be corrected. Thanks!

Reply:
The `attributes`, `colorGrouping`, `measurements`, `sizeguides`, `sizes`, `grouping`, `groupingProperties` reducers have similar logics.

Reply:
Changed.